qiskit/utils/backend_utils.py

```diff
@@ -34,6 +34,23 @@ def __init__(self) -> None:
 _PROVIDER_CHECK = ProviderCheck()
 
 
+def _get_backend_version(backend):
```

Comment:
Related to the comment above about backend version, we should distinguish the two backend version. If agree to refer to call 2 in BackendV2 the `backend_interface_version`,  can we change the method name to `_get_backend_interface_version`?

Reply:
Yes, although something shorter than `backend_interface_version` might be better. Whatever it is, should be used uniformly.

I don't know how urgent this PR is. If we are not making API changes, then we might be able to clean up the language in another PR.

Reply:
This PR is blocking the [0.19.2](https://github.com/Qiskit/qiskit-terra/milestone/21) release as it's necessary to fix this before the ibm provider and aqt provider adopt the v2 interface. You can see PRs for this in progress here:

https://github.com/Qiskit-Partners/qiskit-aqt-provider/pull/51
https://github.com/Qiskit/qiskit-ibm-runtime/pull/102

So it'd be good to get this PR in sooner rather than later. At this point in the week though we probably won't have the release ready until some point next week. I'll update the method name

Reply:
Done in: 363e8dd